NGitLab.Tests/Docker/GitLabTestContext.cs

```diff
@@ -180,6 +185,30 @@ void AddSomeCommits()
                         }));
                 }
             }
+
+            void AddMergeRequest()
```

Comment:
You can already use `CreateMergeRequest()`. I don't think we need a new method.

Reply:
Done.